package.json

```diff
@@ -24,7 +24,7 @@
     "eslint": "4.19.1",
     "eslint-config-google": "0.9.1",
     "lerna": "3.22.1",
-    "mocha": "8.1.2",
+    "mocha": "^5.2.0",
```

Comment:
remove ^

Reply:
why degrade this? from 8.1.2 to 5.2.0

Reply:
Reason for downgrading is due to the issue found here : https://travis-ci.org/github/GovTechSG/mcf-boilerplate-js/jobs/722993061.

![Screenshot 2020-09-07 at 1 00 52 PM](https://user-images.githubusercontent.com/46778490/92350035-5cf8d600-f10a-11ea-85a2-7456ef53993d.png)

This `esm-utils.js` file can only be found on ver 5.2.0.